src/components/Confirmation.vue

```diff
@@ -48,21 +48,20 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 import ClickOutside from 'vue-click-outside'
 
 export default {
-	name: 'PopoverMenu',
```

Comment:
No name?

Reply:
I can add it back, but I think it is not necessary. The directive works and doesn't raise an error in the console. Also, this directive won't be used anymore after #818 is merged.